appveyor.yml

```diff
@@ -15,7 +15,7 @@ test: off
 
 artifacts:
 - path: package\*.nupkg
-#  - path: package\*.msi
+- path: package\*.msi
```

Comment:
I commented this out in the face of uncertainty over whether we would continue to have msi packages. Not a serious issue here, but would be if in our more public feeds.